buildscripts/incremental/setup_conda_environment.sh

```diff
@@ -52,7 +56,7 @@ source activate $CONDA_ENV
 set -v
 
 # Install optional packages into activated env
-if [ "${VANILLA_INSTALL}" != "yes" ]; then
+if [ "$PYTHON" != "3.10" || "${VANILLA_INSTALL}" != "yes" ]; then
```

Comment:
Probably needs to be:
```suggestion
if [ "$PYTHON" != "3.10" ] || [ "${VANILLA_INSTALL}" != "yes" ]; then
```
or
```suggestion
if [[ "$PYTHON" != "3.10" || "${VANILLA_INSTALL}" != "yes" ]]; then
```

Reply:
The solution was:

```
if [[ "$PYTHON" != "3.10" && "${VANILLA_INSTALL}" != "yes" ]]; then
```